org.mixedrealitytoolkit.spatialmanipulation/Solvers/ConstantViewSize.cs

```diff
@@ -153,9 +153,17 @@ public float FovScale
 
         #endregion
 
+        private Vector3 originalScale;
         private float fovScalar = 1f;
         private float objectSize = 1f;
 
+        /// <inheritdoc/>
+        protected override void Awake()
+        {
+            base.Awake();
+            originalScale = GoalScale;
```

Comment:
I don't have enough context on the original intent here.

@keveleigh  or @david-c-kline ,  how do you feel about this change?